presentation/src/main/java/me/androidbox/presentation/alarm_manager/AlarmReminderProvider.kt

```diff
@@ -0,0 +1,39 @@
+package me.androidbox.presentation.alarm_manager
+
+import me.androidbox.component.agenda.AlarmReminderItem
+import java.time.ZoneId
+import java.time.ZonedDateTime
+
+object AlarmReminderProvider {
+
+    fun getRemindAt(
+        alarmReminderItem: AlarmReminderItem,
+        startTime: ZonedDateTime,
+        startDate: ZonedDateTime
```

Comment:
`ZonedDateTime` already contains both date and time as the name says, so only one value is needed here